tableauserverclient/models/schedule_item.py

```diff
@@ -33,6 +33,10 @@ def __init__(self, name, priority, schedule_type, execution_order, interval_item
         self.priority = priority
         self.schedule_type = schedule_type
 
+    def __repr__(self):
+        print(self.__dict__)
+        return "<Schedule#{_id} \"{_name}\" {interval_item}>".format(**self.__dict__)
```

Comment:
#140 filed.  And yeah, I guessed you wouldn't like it 👅
(and oh, I need to remove that print statement!)